scripts/data_validation.py

```diff
@@ -0,0 +1,674 @@
+import os
+import pypsa
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import pycountry
+import logging
+import warnings
+warnings.filterwarnings("ignore")
+
+
+def load_ember_data():
+    """
+    Load the Ember data from a CSV file located in the data folder.
+
+    Returns:
+        pd.DataFrame: Ember data in long format.
+    """
+    ember_data_path = os.path.join(
+        os.getcwd(), "data/validation/ember_yearly_full_release_long_format.csv")
+    ember_data = pd.read_csv(ember_data_path)
+    return ember_data
+
+
+def get_network_path(scenario_folder):
```

Comment:
Hi, @GbotemiB. Thanks for the great PR. It is important to have such validation script. I would suggest moving `get_network_path` into `_helpers.py` script. We might need it in other scripts too. I would also suggest to rename it as `get_solved_network_path` so it is clear that the path is for solved networks, because we might also need in future similar function for prepared network. (In that time maybe we will merge two functions, and give `solved/unsolved` as arguements, but for now it is ok).